sandstone/apps/filebrowser/static/filebrowser.controller.js

```diff
@@ -72,6 +72,16 @@ angular.module('sandstone.filebrowser')
     self.isCopied = true;
   };
 
+  self.openFileInEditor = function() {
+      var message = {
+          key: 'editor',
```

Comment:
@saurabhsood91 Let's namespace events with the ng app name. For example, `editor:openFile`.